bundles/org.openhab.binding.draytonwiser/README.md

```diff
@@ -0,0 +1,248 @@
+2. Enable setup mode on the HeatHub. Connect a machine temporarily to the `WiserHeat_XXXXX` network and browse to `http://192.168.8.1/secret` to obtain the `secret`.
+
+The `refresh` interval defines in seconds, how often the binding will poll the controller for updates.
+
+The `awaySetPoint` defines the temperature in degrees Celsius that will be sent to the heathub when away mode is activated.
+
+## Channels
+
+### Readonly Channels
+
+#### Boiler Controller
+
+| Channel                      | Item Type            | Description                                              |
+|------------------------------|----------------------|----------------------------------------------------------|
+| `heatingOverride`            | Switch               | State of the heating override button on the controller   |
+| `heatChannel1Demand`         | Number:Dimensionless | Current demand level of heating channel 1                |
+| `heatChannel1DemandState`    | Switch               | Is channel 1 calling the boiler for heat                 |
+| `heatChannel2Demand`         | Number:Dimensionless | Current demand level of heating channel 2                |
+| `heatChannel2DemandState`    | Switch               | Is channel 2 calling the boiler for heat                 |
+| `currentSignalRSSI`          | Number               | Relative Signal Strength Indicator                       |
+| `currentWiserSignalStrength` | String               | Human readable signal strength                           |
+| `currentSignalStrength`      | Number               | Signal strength value that maps to qualityofservice icon |
+
+#### Hot Water
+
+| Channel                  | Item Type    | Description                                              |
+|--------------------------|--------------|----------------------------------------------------------|
+| `hotWaterOverride`       | Switch       | State of the hot water override button on the controller |
+| `hotWaterDemandState`    | Switch       | Is hot water calling the boiler for heat                 |
+| `hotWaterBoosted`        | Switch       | Is hot water currently being boosted                     |
+| `hotWaterBoostRemaining` | Number:Time  | How long until the boost deactivates in minutes          |
+
+#### Room
+
+| Channel              | Item Type            | Description                                                                  |
+|----------------------|----------------------|------------------------------------------------------------------------------|
+| `currentTemperature` | Number:Temperature   | Currently reported temperature                                               |
+| `currentHumidity`    | Number:Dimensionless | Currently reported humidity (if there is a room stat configured in this room |
+| `currentDemand`      | Number:Dimensionless | Current heat demand percentage of the room                                   |
```

Comment:
You could use `PercentType`.

Reply:
`PercentType` is a dimmer. If the channel is readonly it's generally prefered to not use a dimmer, but use the `Number:Dimensionless`

Reply:
Oops, mixed that up, sorry.